data-rights-protocol.md

```diff
@@ -129,11 +134,11 @@ The request body MUST adhere to the [Exercise Status Schema](#303-schema-status-
 
 #### 2.04.1 `POST $status_callback` Response
 
-Covered Business SHOULD make a best effort to ensure that a 200 status is recorded for the most recent status update. The body of the callback's response SHOULD be discarded and not be considered for parsing by the Covered Business.
+Privacy Infrastructure Providers SHOULD make a best effort to ensure that a 200 status is recorded for the most recent status update. The body of the callback's response SHOULD be discarded and not be considered for parsing by the Covered Business.
```

Comment:
What does it mean for a `200` status to be recorded?  I'm not sure I'm following what's meant here.